tests/API/XCCDF/unittests/all.sh

```diff
@@ -2,9 +2,12 @@
 
 set -e -o pipefail
 
-. ../../../test_common.sh
+. $builddir/tests/test_common.sh
 
 test_init test_api_xccdf_unittests.log
+
+touch not_executable
```

Comment:
?

Reply:
@mpreisler 
This file (`not_executable`) is needed for some tests. It is referenced in some ovals we use, eg. in https://github.com/OpenSCAP/openscap/blob/maint-1.2/tests/API/XCCDF/unittests/oval/fail/oval.xml#L49.

 In automake build system, this file was created by a Makefile. See https://github.com/OpenSCAP/openscap/blob/maint-1.2/tests/API/XCCDF/unittests/Makefile.am#L24
I find creating files as unusual usage of build system. I decided to create the file in the related test script, because we already create test files this way in other tests.